features/features.xml

```diff
@@ -22,6 +22,7 @@
     <test testName="asm_cp" configFile="config/TEMPLATE_asm.xml" />
     <test testName="asm_st" configFile="config/TEMPLATE_asm.xml" />
     <test testName="asm_ld" configFile="config/TEMPLATE_asm.xml" />
+    <test testName="asm_ldmatrix" configFile="config/TEMPLATE_asm.xml" />
```

Comment:
already in https://github.com/oneapi-src/SYCLomatic-test/pull/908

Reply:
Removed it